website/docs/reference/operators/array-operators.md

```diff
@@ -2,4 +2,197 @@
 sidebar_position: 3
 ---
 
-# Array operators
+# Array query operators
+
+Array query operators allow you to search for specific elements within an array field in a document.
+
+|Operator|Description|
+|-------|-----------|
+|[`$all`](#all)|Matches an array that contains all the elements in the specified query|
+|[`$elemMatch`](#elemmatch)|Matches a document that contains an array field with at least one element that matches all the specified query criteria|
+|[`$size`](#size)|Matches an array with a specified number of elements|
+
+For the examples in this section, insert the following documents into the `team` collection:
+
+```js
+db.team.insertMany([
+   {
+      id: 1,
+      name: "Jack Smith",
+      position: "Manager",
+      skills: ["leadership", "communication", "project management"],
+      contact: {
+         email: "john@example.com",
+         phone: "123-456-7890"
+      },
+      active: true
+   },
+   {
+      id: 2,
+      name: "Jane Mark",
+      position: "Software Developer",
+      skills: ["Java", "Python", "C++"],
+      contact: {
+         email: "jane@example.com",
+         phone: "123-456-7891"
+      },
+      active: false
+   },
+   {
+      id: 3,
+      name: "Bob Johnson",
+      position: "Graphic Designer",
+      skills: ["Adobe Photoshop", "Illustrator", "InDesign"],
+      contact: {
+         email: "bob@example.com",
+         phone: "123-456-7892"
+      },
+      active: true
+   },
+   {
+      id: 4,
+      name: "Alice Williams",
+      position: "Marketing Coordinator",
+      skills: ["communication", "content creation", "event planning"],
+      contact: {
+         email: "alice@example.com",
+         phone: "123-456-7893"
+      },
+      active: true
+   }
+])
+```
+
+## $all
+
+*Syntax*: `{ <field>: { $all: [ <value1>, <value2>, ... ] } }`
+
+The `$all` operator matches documents where the field value is an array that contains all the specified elements.
+
+For example, find all documents in the `team` collection where the `skills` field contains a `communication` element using the following query operation:
+
+```js
+db.team.find({
+   "skills": {
+      $all: ["communication"]
+   }
+})
```

Comment:
This example does not highlight the difference of `$all` from `$in`. I think it might be a bit more clear if we have more than 1 element in the array query. What do you think?